tests/server/api/v1/test_records.py

```diff
@@ -145,3 +158,69 @@ def test_create_record_response_with_nonexistent_record_id(client: TestClient, d
 
     assert response.status_code == 404
     assert db.query(Response).count() == 0
+
+
+def test_list_record_responses(client: TestClient, db: Session, admin: User, admin_auth_header: dict):
+    record = RecordFactory.create()
+    response_a = ResponseFactory.create(record=record)
+    response_b = ResponseFactory.create(record=record)
+
+    response = client.get(f"/api/v1/records/{record.id}/responses", headers=admin_auth_header)
+
+    assert response.status_code == 200
+    assert response.json() == {
+        "items": [
+            {
+                "id": str(response_a.id),
+                "inserted_at": response_a.inserted_at.isoformat(),
+                "updated_at": response_a.updated_at.isoformat(),
+                "values": response_a.values,
+                "user_id": str(response_a.user_id),
```

Comment:
Let's follow model columns order (only for making it easier to read):

```suggestion
                "id": str(response_a.id),
                "values": response_a.values,
                "user_id": str(response_a.user_id),
                "inserted_at": response_a.inserted_at.isoformat(),
                "updated_at": response_a.updated_at.isoformat(),
```